src/utils.ts

```diff
@@ -378,14 +378,16 @@ const publishPromptTemplate = async (body: PublishPromptTemplate) => {
 };
 
 const cleaned_result = (results: any[]) => {
```

Comment:
@buckyroberts can you also provide an example of how and where these changes will be applicable? 
Also, I think we should just catch all the streaming responses and send them to the server for post processing. JavaScript sdk mimics python sdk and I believe the same issue would be present there as well.